### PR TITLE
add name: PR (Angel)

### PR DESCRIPTION
**Pull request by angelitora**

### CODE REVIEW

**Review by soerenthomsen — APPROVED** [review]

**Comment by soerenthomsen** [comment]
Let' see who is fastest with providing a review! We always need two official approvals before I can merge :)

**Review by abr099 — APPROVED** [review]

File: training_session_07.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ Date: 20/06/2022, 16:00 CEST
 - [Ailin Brakstad](https://github.com/abr099)
 - [Clara M. Loureiro](http://github.com/cangelaml)
 - [Anthony Bosse](https://github.com/AnthonyBosse)
+- [Angel Ruiz-Angulo](https://github.com/angelitora)
+
```
Comment (hassanimaryam) [inline training_session_07.md:21]: - [Maryam Hassani] (https://github.com/hassanimaryam)

**Review by hassanimaryam — APPROVED** [review]

File: training_session_07.md (hunk 1 above)
Comment (hassanimaryam) [inline training_session_07.md:21]: - [Maryam Hassani](https://github.com/hassanimaryam)

Comment (hassanimaryam) [inline training_session_07.md:21]: - [Maryam Hassani](https://github.com/hassanimaryam)

**Comment by soerenthomsen** [comment]
Thanks @angelitora for this PR and @abr099 and @hassanimaryam for reviewing it :)

I will merge it now!

File: training_session_07.md (hunk 1 above)
Comment (hassanimaryam) [inline training_session_07.md:21]: - [Maryam Hassani](https://github.com/hassanimaryam)

File: training_session_07.md [hunk 2]
```diff
@@ -17,6 +17,8 @@ Date: 20/06/2022, 16:00 CEST
...
```
Comment (hassanimaryam) [inline training_session_07.md:22]: -  [Maryam Hassani](https://github.com/hassanimaryam)

File: training_session_07.md (hunk 1 above)
Comment (hassanimaryam) [inline training_session_07.md:21]: [Maryam Hassani](https://github.com/hassanimaryam)

**Pull request merged**